assets/js/system-status.ts

```diff
@@ -0,0 +1,120 @@
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
This was copied and pasted from https://github.com/maxmind/blog-site/blob/b81807f3185bf128482bb4db38abef3f61ccd3c6/assets/js/system-status.ts#L1-L120.

It is also the same code as https://github.com/maxmind-zendesk-theme/zendesk-theme/blob/main/src/scripts/ui/system-status.js.

Reply:
Cool! Might be a good idea to create a follow-up linear issue to de-duplicate this code across the 3 sites. Maybe a private npm package or something?